server/plugin/plugin_test.go

```diff
@@ -93,12 +105,31 @@ func TestPluginOnActivate(t *testing.T) {
 			}
 		})
 	}
+	t.Run("NewStore() fails", func(t *testing.T) {
+		api := &plugintest.API{}
+		api.On("GetServerVersion").Return(minimumServerVersion)
+		defer api.AssertExpectations(t)
+
+		patch := monkey.Patch(kvstore.NewStore, func(plugin.API) (store.Store, error) {
```

Comment:
I know this is ugly, but I didn't find another way to properly test this

Reply:
Would you like to add `UpdateDatabase` func to `store.Store` interface? If doing so, we can modify `NewStore` func to return error by mocking `UpdateDatabase`.
I do not feel strange even if that `UpdateDatabase` func is in the `store.Store` interface.

How does it sounds? If you agree my idea, see also [this comments](https://github.com/matterpoll/matterpoll/pull/85#discussion_r228826995).

Reply:
Sure we can add `UpdateDatabase` to `store.Store`. But how does this helps us for this problem? `kvstore.NewStore` will always return a `kvstore.Store`.
I don't think I get what you are up to.

Reply:
Ah..sorry. I misunderstood that we can mock `UpdateDatabase` if adding it to interface...
But we can't do that. you are right.

Now I don't have a better solutions. So I'm fine with present code.